kitodo-mediaserver-cli/src/main/java/org/kitodo/mediaserver/cli/CliApplication.java

```diff
@@ -26,13 +28,39 @@
      */
     public static void main(String[] args) {
 
+        String appDir = getApplicationDirectory();
+
         new SpringApplicationBuilder(CliApplication.class)
             .properties(
                 "spring.config.name:"
                     + "default,"
                     + "local,"
                     + "application,"
-                    + "dev")
+                    + "dev",
+                "spring.config.additional-location:" + appDir + "/," + appDir + "/config/")
             .build().run(args);
     }
+
+    /**
+     * Get the directory where this application is located.
+     *
+     * @return file system path
+     */
+    private static String getApplicationDirectory() {
+
+        String path = CliApplication.class.getProtectionDomain().getCodeSource().getLocation().getPath();
+
+        // If we are in a JAR file, remove the part from inside the JAR
+        int splitterPos = path.indexOf("!");
+        if (splitterPos >= 0) {
+            path = path.substring(0, splitterPos);
+        }
```

Comment:
You don't wish to use the simpler version here?
path = path.split("!")[0];

Reply:
Oh, of course! :-)